src/frontend/shortlist/src/components/layout/shortlists/ShortlistManager.vue

```diff
@@ -5,7 +5,7 @@ import ShortlistRow from "./ShortlistRow.vue";
 export default {
   name: "ShortlistManager",
   props: ["studentShortlists"],
-  components: { ShortlistRow },
+  components: { ShortlistRow},
```

Comment:
Are these commits in ShortlistManager.vue necessary?

Reply:
> Are these commits in ShortlistManager.vue necessary?

I had to switch branch at that moment. Git does not allow switching without commiting.

Reply:
> > The style looks great as I checked it out however there's a question: Something I want to be sure, so before the color of the list is in listSettings.color, which sits in a json file currently you are using localStorage.getItem. Is this gonna get the color setting from the json file? Because in color switch I did not see an attempt to move back up to view.vue to make change to the json color settings (like $emit), I think I would also like to see Daniel's review on this change to be sure since he set up the new structure
> 
> I am not sure what you meant by adding things to json file. Is it necessary to save a user's taste in lists' colors into a database? I am also not sure about what you meant by "cache" you mentioned in a private message. I thought we should not worry about caching at this stage of development.

Then what is the "localStorage" you used, isn't it saving the data to browser temporarily? Also for any of you guys who is not sure what goes in the data base, check js files in the shortlist/api. They give a clear description of what's in db to use